### ISSUE TITLE
Ethanol reduces manifestations

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#4796]
**Preferred term label:**

Amerliorated by ethanol ingestion
**Synonyms**

Ethanol reduces manifestations
**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal adding as HP:0032503
